Docker/StorageApi/Writer.php

```diff
@@ -444,7 +447,14 @@ protected function uploadTable($source, $config = array())
                 $options["columns"] = $config["columns"];
                 $options["withoutHeaders"] = true;
             }
-            $this->client->writeTableAsync($config["destination"], $csvFile, $options);
+            if (is_dir($source)) {
```

Comment:
ne, protože tohle možná patří do storage api klienta? nebo do nějakýho wrapperu okolo něj nebo do samostatnýho balíčku, co bude řešit data-loader container.